convert/src/mongo.rs

```diff
@@ -0,0 +1,14 @@
+/// Convert MongoDB timestamps to Rust's SystemTime
+pub fn timestamp_to_systime(ts: bson::Timestamp) -> std::time::SystemTime {
+    std::time::UNIX_EPOCH + std::time::Duration::from_secs(ts.time.into())
+}
+
+/// Convert a SystemTime to a MongoDB timestamp. Note that MongoDB sucks and stores unix timestamps in 32 bits
```

Comment:
I would remove the "mongo sucks" part of the comment. I think that is actually a reasonable design choice:
in 2106, when this will stop working, Mongo is likely not going to be around anymore so why wasting bits to store longer timestamps

Reply:
Unix now uses 64 bits, so that’s why I don’t agree with their design choice

Reply:
fair enough, it has its trade-offs, but I guess they decided that for a DB is more important to save bits rather than be super consistent with the rest of the world